satip/eumetsat.py

```diff
@@ -198,19 +217,22 @@ def json_extract(json_obj: Union[dict, list], locators: list):
     return extracted_obj
 
 
+# TODO: Remove this class if `check_valid_requests` is removed as well.
+class InvalidCredentials(Exception):
+    """Class to catch the case of invalid credentials."""
+
+    pass
+
+
+# TODO: Remove this function, as its only functionality can also be achieved with raise_for_status
```

Comment:
Same here, feel free to remove it! Less code is always better, and there has been a few things left from a large refactoring a few months ago.